### PR TITLE
making toggle selection darker

### LINKED ISSUE #477: improve selected toggle design
originally a comment from @RoniRos regarding testing of v1.8

https://delphi-org.slack.com/archives/C011415D2CE/p1598493232003900

> In two-tone toggles, it’s not immediately clear which tone is “on”. Consider a different affordance method, like a frame around the active mode?

current dev: 
image

current stable:

image

### PR DESCRIPTION
**Pull request by kateharwood**
closes #477 

**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary
Makes the background darker on the toggle buttons "Incidence" vs "Cumulative" and "Counts" vs "Ratios" when selected

Before:
Screen Shot 2020-10-14 at 4 02 16 PM

After:
Screen Shot 2020-10-14 at 4 02 22 PM

### CODE REVIEW

**Review by tildechris — COMMENTED** [review]
LGTM, but we should get UX feedback before approval/merge.

**Review by sgratzl — COMMENTED** [review]
I wonder what the contrast between the black font and the dark background is. We had a bunch of accessibility issues regarding that, e.g., the map title

**Comment by kateharwood** [comment]
Good call, I just checked and the contrast passes accessibility guidelines.

**Review by sgratzl — APPROVED** [review]

**Comment by kateharwood** [comment]
Either @Letsgohyde @philmcguinn or @jennyxxdance looking for a quick UX pass before I can merge.

**Comment by philmcguinn** [comment]
LGTM!

**Pull request merged**
